packages/augur-ui/src/modules/reporting/common.tsx

```diff
@@ -1222,11 +1221,12 @@ export const ParticipationTokensView = (
           "The total amount of unclaimed Dai you've earned through reporting"
         }
       />
-
-      <PrimaryButton
+      <ProcessingButton
         disabled={!hasRedeemable}
```

Comment:
disabling is off, the button should be disabled if there is a pending queue entry with status `pending`